firefox-ios/firefox-ios-tests/Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -33,8 +33,7 @@ class ActivityStreamTest: BaseTestCase {
                                LaunchArguments.DisableAnimations]
         }
         launchArguments.append(LaunchArguments.SkipAddingGoogleTopSite)
-        // Comment out so that the sponsors shortcuts are shown after the homepage redesign. (FXIOS-11668)
-        // launchArguments.append(LaunchArguments.SkipSponsoredShortcuts)
+        launchArguments.append(LaunchArguments.SkipSponsoredShortcuts)
```

Comment:
@clarmso Can you please verify that these tests no longer fail for you on this branch as well?